docs/installation.md

```diff
@@ -215,77 +213,7 @@ sumologic:
   clusterName: ${MY_CLUSTER_NAME}
 ```
 
-#### OpenShift with limiting the scope of the interaction of our Prometheus Operator
-
-An example configuration for Openshift which install our Prometheus Operator side by side with an existing one:
-
-```yaml
-sumologic:
-  accessId: ${SUMO_ACCESS_ID}
-  accessKey: ${SUMO_ACCESS_KEY}
-  clusterName: ${MY_CLUSTER_NAME}
-  scc:
-    create: true
-kube-prometheus-stack:
-  prometheusOperator:
-    namespaces:
-      additional:
-        - my-namespace
-  prometheus-node-exporter:
-    service:
-      port: 9200
-      targetPort: 9200
-otellogs:
-  daemonset:
-    containers:
-      otelcol:
-        securityContext:
-          privileged: true
-    initContainers:
-      changeowner:
-        securityContext:
-          privileged: true
-tailing-sidecar-operator:
-  scc:
-    create: true
-```
-
-#### OpenShift using existing Prometheus Operator which is by default available in `openshift-monitoring` namespace
-
-An example configuration for OpenShift which uses existing Prometheus Operator:
-
-```yaml
-sumologic:
-  accessId: ${SUMO_ACCESS_ID}
-  accessKey: ${SUMO_ACCESS_KEY}
-  clusterName: ${MY_CLUSTER_NAME}
-  scc:
-    create: true
-kube-prometheus-stack:
-  prometheus-node-exporter:
-    service:
-      port: 9200
-      targetPort: 9200
-  prometheusOperator:
-    enabled: false
-otellogs:
-  daemonset:
-    containers:
-      otelcol:
-        securityContext:
-          privileged: true
-    initContainers:
-      changeowner:
-        securityContext:
-          privileged: true
-tailing-sidecar-operator:
-  scc:
-    create: true
-```
-
-**NOTE:** Please refer to
-[Using existing Operator to create Sumo Logic Prometheus instance](/docs/prometheus.md#using-existing-operator-to-create-sumo-logic-prometheus-instance)
-before applying the configuration.
+More configuration examples can be seen in [this document](/docs/configuration-examples.md).
```

Comment:
Could we link the Openshift ones? As we don't have separate instruction for OpenShift, I think we should specifically say that Openshift examples are in examples

Reply:
You mean this one?
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/28b78fb8b5c0d19fb28cfd9376b280635c56bfc5/docs/installation.md#installing-in-openshift-platform

Reply:
Example was composition of few sections from installation section

Reply:
basically configuration you linked and prometheus one's, but it is not obvious for me that you need to go to examples to find complete openshift configuration

Reply:
or linking examples to openshift in `openshift installation` section

Reply:
I did the first one.